src/uniparc/components/entry/Entry.tsx

```diff
@@ -78,7 +95,7 @@ const Entry: FC = () => {
 
   const transformedData = uniParcConverter(dataObject.data);
 
-  const entrySidebar = <XRefsFacets xrefs={dataObject} />;
+  const entrySidebar = <XRefsFacets xrefs={xrefsDataObject} />;
```

Comment:
Noticed that the Organisms facets disappear:
1. http://localhost:8080/uniparc/UPI000000000B/entry
2. Choose an organism from the Organisms facet
3. The Organisms facet will disappear

Reply:
🤔 mmh, I can't really reproduce on my side.
If you want ping me whenever you want on Slack and we can go over this specific issue

Reply:
Should be fixed now